backend/docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: '3.2'
+
+services:
+  mysql-docker-container:
+    image: mysql/mysql-server:8.0.13
+    restart: always
+    environment:
+      - MYSQL_ROOT_PASSWORD=scripty
+      - MYSQL_DATABASE=inventorsoftacademy
+      - MYSQL_USER=scripty
+      - MYSQL_PASSWORD=scripty
+    volumes:
+      - mysql-db:/d/db/data

```

Comment:
Right side for the volume is path inside container. Container is based on linux so I doubt that path like `/d/db/data` exists inside container. Docker will create those folders for you but mysql doesn't know anything about it. It will store db data inside `/var/lib/mysql` directory, according to the docs